### PR TITLE
feat(dashboard): render users

### PR DESCRIPTION
**Pull request by moalidv**

### CODE REVIEW

**Comment by moalidv** [comment]
I pushed this branch from my local on (mo/dashboard-users) which was branched originally from master 
I don't know why all these files and changes appear here in this PR.

File: apps/dashboard/src/components/Users/List.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        header: intl("dashboard.user.email"),
+        cell: (info) => {
+          return info.getValue();
+        },
+      }),
+      columnHelper.accessor("password", {
+        header: intl("dashboard.user.hasPassword"),
+        cell: (info) => {
+          return <BooleanField checked={info.row.original.password} />;
+        },
```
Comment (neuodev) [inline apps/dashboard/src/components/Users/List.tsx:58]:
```suggestion
        cell: (info) => <BooleanField checked={info.row.original.password} />
```

File: apps/dashboard/src/components/Users/List.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                },
+              ]}
+            />
+          ),
+        }),
+      ] as Array<ColumnDef<IUser.Self, unknown>>,
+    [columnHelper]
+  );
+  if (query.isLoading) return <Loading className="h-1/4" />;
+  if (!query.data) return null;
```
[the remark this answers is not in the document]
Comment (neuodev) [reply apps/dashboard/src/components/Users/List.tsx:151]: @moalidv please add the error element

File: apps/dashboard/src/pages/Root.tsx [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
-import { Outlet } from "react-router-dom";
+import { Outlet, useNavigate } from "react-router-dom";
+import { useAppSelector } from "@/redux/store";
+import { profileSelectors } from "@/redux/user/profile";
+import { IUser } from "@litespace/types";
+import { Route } from "@/lib/route";
 
 const Root: React.FC = () => {
+  const profile = useAppSelector(profileSelectors.user);
+  const navigate = useNavigate();
```
Comment (neuodev) [inline apps/dashboard/src/pages/Root.tsx:11]: Please add empty line. 
```suggestion
  const navigate = useNavigate();
  
```

File: apps/dashboard/src/pages/Users.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          title={intl("dashboard.users.title")}
+          fetching={users.query.isFetching && !users.query.isLoading}
+          count={users.query.data?.total}
+        />
+        {/* <div className="flex flex-row items-center gap-4">
+          <h1 className="text-2xl">{intl("dashboard.users.title")}</h1>
+          {users.query.isFetching && !users.query.isLoading ? (
+            <Spinner />
+          ) : null}
+        </div> */}
```
Comment (neuodev) [inline apps/dashboard/src/pages/Users.tsx:24]: Left over.

```suggestion
```

File: packages/luna/src/locales/ar-eg.json [hunk 5]
```diff
@@ -243,6 +244,20 @@
   "dashboard.plan.delete.error": "نأسف، حدث خطأ أثناء حذف الخطة",
   "dashboard.plan.form.update.success": "تم تعديل الخطة بنجاح",
   "dashboard.plan.form.update.error": "نأسف، حدث خطأ أثناء تعديل الخطة",
+  "dashboard.users.title": "المستخدمين",
+  "dashboard.user.name": "الاسم",
+  "dashboard.user.email": "الإيميل",
+  "dashboard.user.hasPassword": "يمتلك رقم سري",
+  "dashboard.user.fullMonthPrice": "السعر الشهري",
```
Comment (neuodev) [inline packages/luna/src/locales/ar-eg.json:251]: There is no such thing as `fullMonthPrice` for the user. 
```suggestion
```

**Review by neuodev — CHANGES_REQUESTED** [review]
Great work @moalidv! Please take a look at my minor comments above! Thanks in advance!

**Review by neuodev — APPROVED** [review]

**Pull request merged**